weave/Scripts/Main.cs

```diff
@@ -89,6 +82,7 @@ private void SpawnPlayers()
             _players.Add(player);
 
             AddChild(player);
+            player.CurveSpawner.CreatedLine += HandleDrawLine;
```

Comment:
Not needed to change, but its often handy to one line and just do

player.CurveSpawner.CreatedLine += (line) => AddChild(line);

or however it could be written